### ISSUE TITLE
Confirm kick distributions for players are correct

### ISSUE THREAD

**Issue by marlitas** [phetsims/center-and-variability#246]
From https://github.com/phetsims/center-and-variability/issues/235, I don't feel confident enough to determine wether `Individual players have correct kick distributions` is true or not. This feels like something @catherinecarter should confirm, so that we can adjust distributions in code if needed.

**Comment by catherinecarter** [comment]
I looked at many distributions, and I don't think the distributions were disparate enough to create the desired distributions. I've updated them in [this spreadsheet](https://docs.google.com/spreadsheets/d/1GM8a0JVuV46Mm_ieF8jUKhatj2qU9nIegG9t_LE8ZyY/edit#gid=1398788261). Perhaps these more extreme distributions will do the trick.

**Comment by catherinecarter** [comment]
I tested the distributions. Kickers 1 and 2 look great. Kickers 3 and 4 looked off, so I updated the distributions in the spreadsheet to reflect a more obvious skew. Let's see how those look.

**Comment by samreid** [comment]
The skew distribution is supposed to also affect screen 1 and 2 right?

**Comment by catherinecarter** [comment]
@sam, yes, the skew can use the same distribution for screens 1 and 2.

**Comment by samreid** [comment]
OK I updated the skew distributions on screen 1, 2 and for players 3-4 on screen 3.  Ready for review.

**Comment by catherinecarter** [comment]
Thanks, @samreid, the distributions looks much better. I still think there are a lot of kicks on the values towards the skew compared to their probabilities, but overall the distributions look good and the direction of their skew is obvious, even with 15 kicks. I found the skew was most obvious around 20 kicks or more, so I'm glad we added that preference. 

I think the distributions are good enough to call it good and close when ready. thanks!
